src/components/layout/header-scrolling.js

```diff
@@ -39,19 +28,20 @@ const propTypes = {
 class HeaderScrolling extends Component {
     constructor(props) {
         super(props);
-        const {size} = props;
-        const storeState = this._getStateFromStore();
-        storeState.size = size;
+        let storeState = this._getStateFromStore()
```

Comment:
`const` does not mean it will not mutate it means it will not be set to another value so feel free to use it.